tests/test_dataset.py

```diff
@@ -176,8 +178,11 @@ def check_is_expected_response(response):
 
     # Plain image upload
     ds_items_plain = []
-    for url in TEST_IMG_URLS:
-        ds_items_plain.append(DatasetItem(image_location=url))
+    for i, url in enumerate(TEST_IMG_URLS):
+        upload_to_scale = i == 0
```

Comment:
useful to leave an explanatory comment here, i.e. (upload first item in privacy mode)